src/components/LoginRegister/RegisterInput.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { useApolloClient } from '@apollo/client';
+import { useDispatch } from 'react-redux';
+import { Flex, Input, Button, Heading, VStack } from '@chakra-ui/react';
+import { register } from '../../data/actions/auth';
+
+const RegisterInput = () => {
+  const client = useApolloClient();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const dispatch = useDispatch();
+
+  const handleClick = async e => {
+    e.preventDefault();
+    dispatch(register(firstName, lastName, email, password, client));
```

Comment:
Not sure how much work this is but after registering successfully we should redirect to the home page? if too much work do in another PR 

also a simple error message handler like the comment for login component

Reply:
Done!